components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss

```diff
@@ -210,11 +210,42 @@ chef-alert {
   }
 
   .healthcheck-content {
+    position: relative;
     border-radius: 0 0 2px 2px;
     background-color: var(--chef-primary-dark);
     color: var(--chef-white);
-    padding: 8px;
+    padding: 8px 8px 40px 8px;
     overflow-wrap: break-word;
+    max-height: 94px;
```

Comment:
By default, we're setting the max height to 94px... this is 3 lines of output plus the expand button.